codegen/src/test/resources/software/amazon/awssdk/codegen/poet/transform/operationwithnoinputoroutputrequestmodelmarshaller.java

```diff
@@ -0,0 +1,37 @@
+package software.amazon.awssdk.services.jsonprotocoltests.transform;
+
+import javax.annotation.Generated;
+import software.amazon.awssdk.annotations.SdkInternalApi;
+import software.amazon.awssdk.core.exception.SdkClientException;
+import software.amazon.awssdk.core.protocol.ProtocolMarshaller;
+import software.amazon.awssdk.services.jsonprotocoltests.model.OperationWithNoInputOrOutputRequest;
+import software.amazon.awssdk.utils.Validate;
+
+/**
+ * {@link OperationWithNoInputOrOutputRequest} Marshaller
+ */
+@Generated("software.amazon.awssdk:codegen")
+@SdkInternalApi
+public class OperationWithNoInputOrOutputRequestModelMarshaller {
+    private static final OperationWithNoInputOrOutputRequestModelMarshaller INSTANCE = new OperationWithNoInputOrOutputRequestModelMarshaller();
+
+    private OperationWithNoInputOrOutputRequestModelMarshaller() {
+    }
+
+    public static OperationWithNoInputOrOutputRequestModelMarshaller getInstance() {
+        return INSTANCE;
+    }
+
+    /**
+     * Marshall the given parameter object
+     */
+    public void marshall(OperationWithNoInputOrOutputRequest operationWithNoInputOrOutputRequest,
+                         ProtocolMarshaller protocolMarshaller) {
+        Validate.paramNotNull(operationWithNoInputOrOutputRequest, "operationWithNoInputOrOutputRequest");
+        Validate.paramNotNull(protocolMarshaller, "protocolMarshaller");
+        try {
```

Comment:
Be nice to get rid of this if it's easy to do.

Reply:
good point